.mergify.yml

```diff
@@ -0,0 +1,11 @@
+pull_request_rules:
+- actions:
+    merge:
+      method: squash
```

Comment:
Do you also need to add `strict: true`? (https://doc.mergify.io/strict-workflow.html)